### ISSUE TITLE
NMMNH Paleo - unidentifiable IDs that can be upgraded

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/data-migration#252]
@Nicole-Ridgwell-NMMNHS I was doing some checking on stuff today and ran across [this record](https://arctos.database.museum/guid/NMMNH:Paleo:3687). The part is a coprolite, so it's not a plant! Should we add a new ID of Animalia? If so, who should I put as the identifier? I would put "identification made based on identification of part" or something like that in the ID remark.

If this is something you would like to do, we can wait until everything is in, then find all the "unidentifiable" identifications with parts that clearly indicate whether the specimen is a plant or animal and make an identification bulkload file to create the new IDs. (Probably the easiest way to upgrade stuff).

**Comment by campmlc** [comment]
Can't these be treated as we treat cultural objects, e.g. identification =
coprolite in the same way we'd have identification = basket? And then add
animalia?

**Comment by Jegelewicz** [comment]
We don't need that - coprolite is the part name.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Yes, for things that can get an ID from the part name, we should. You can put yourself as the identifier with "identification based on part" in the remark. I'm ok with doing this as a bulkload later.
